packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -390,14 +391,29 @@ const GroupForm = ({
               ))}
 
               {requirementSubForms.length < MAX_REQUIREMENTS && (
-                <CWButton
-                  type="button"
-                  label="Add requirement"
-                  iconLeft="plus"
-                  buttonWidth="full"
-                  buttonType="secondary"
-                  buttonHeight="med"
-                  onClick={addRequirementSubForm}
+                <CWTooltip
```

Comment:
I think this will not show up, as per the check above `requirementSubForms.length < MAX_REQUIREMENTS && (`